src/type/Call.ts

```diff
@@ -57,14 +58,22 @@ export interface NativeCallQualityWarningsEvent {
   previousWarnings: NativeCallQualityWarnings;
 }
 
+export interface NativeCallMessageReceivedEvent {
+  type: Constants.CallEventMessageReceived;
+  call: NativeCallInfo;
+  callMessageSID: string;
```

Comment:
We should reference the common constants for keys of native interfaces, like so:

```
[Constants.CallMessageSID]: string;
```

Reply:
Do you want to create a ticket to continue the pattern through the rest of the files?

Yes, will remove the outer `callMessageSID`

Reply:
Yeah, here's the ticket: https://issues.corp.twilio.com/browse/VBLOCKS-2643